pkg/gcv/validator.go

```diff
@@ -86,9 +90,14 @@ func NewValidatorConfig(policyPaths []string, policyLibraryPath string) (*config
 func newCFClient(
 	targetHandler cfclient.TargetHandler,
 	templates []*cftemplates.ConstraintTemplate,
-	constraints []*unstructured.Unstructured) (
+	constraints []*unstructured.Unstructured,
+	options *InitOptions) (
```

Comment:
structure this as a standard Go option

Reply:
turned InitOptions into an internal struct to hold CF functional opts and added another functional opt type to configure initOptions. lmk what you think.